new-frontiers-14/frontier-station-14
# Searchlight Air-Alarm Fix

## About the PR


The searchlight has an incorrect air alarm configuration. This fixes it by adding the missing scrubber/vent to the network configuration.

## Why / Balance

Currently the air alarm can never tell what the air quality is like in the main room of the searchlight, and the fire locks will never trigger. Also it's nice when ships don't come with issues from the shipyard.

## How to test

Buy a searchlight in the current version, click on the air alarm next to the airlock, you'll notice there's a missing air vent and scrubber.

## Media



**Changelog**

:cl:
- fix: connected air vent and scrubber to the rear air alarm in the searchlight.
